packages/syntax-core/src/Badge/Badge.module.css

```diff
@@ -0,0 +1,5 @@
+.badge {
+  border-radius: 100px;
+  padding: 4px 8px;
+  display: inline-block;
```

Comment:
@christianvuerings If I don't do some sort of `inline` display then the badge takes up the entire width available instead of just the width of the text plus padding

Reply:
@jliotta yes, should be fine - thank you